automated/health-checks/tests/health-check-test.yaml

```diff
@@ -0,0 +1,26 @@
+metadata:
+    name: Test Health Check
+    format: "Lava-Test Test Definition 1.0"
+    description: "Standard Health Check for RevPi devices"
+    maintainer:
+        - r.gsponer@kunbus.com
+    os:
+        - raspian
+    scope:
+        - functional
+    devices:
+        - Core
+        - Core 3
+        - core 3+
+        - Core S
+        - Core SE
+        - Connect
+        - Connect+
+        - Connect S
+        - Connect SE
+        - Compact
+        - Flat
+run:
+    steps:
+        - cd automated/health-checks/tests
+        - bash ./health-check.sh "${DUT}"
```

Comment:
What's DUT used for? I cannot the any use of $1 in the health check command

Reply:
@RamiGspo What was the idea here? I'm not sure